tester/main.cpp

```diff
@@ -1,5 +1,5 @@
 #include "versioningCompiler/Version.hpp"
-
+#include <vector>
```

Comment:
If you remove the `opt_vec_t` and replace it with the list-equivalent version, this one should be not required anymore.

Reply:
Changed everything in opt_list_t